.travis.yml

```diff
@@ -1,33 +1,32 @@
-language: rust
 dist: trusty
 sudo: required
-rust:
-- beta
-- nightly
-- stable
-
-cache:
-  directories:
-  - target
+language: rust
 
 os:
   - linux
 
+rust:
+  - stable
+  - beta
+  - nightly
+
 matrix:
   include:
   - os: osx
```

Comment:
how about remove this metrics and add osx line 6?

Reply:
Just keep consistent with `master` branch